src/main/java/hudson/plugins/git/extensions/GitSCMExtension.java

```diff
@@ -352,6 +352,15 @@ public GitClientType getRequiredClient() {
         return GitClientType.ANY;
     }
 
+    /**
+     *
+     * @return <code>true</code> to disable the scheduling of another build to catch up
+     * when multiple revisions are detected
+     */
+    public boolean enableMultipleRevisionDetection() {
+        return true;
```

Comment:
IIUC, this will make the default behaviour as the previous one to this PR, won't it?

Reply:
Yes this ensures that the default behaviour does not change as long as the new extension is not selected in the job config.